examples/tensorrt_llm/common/base_engine.py

```diff
@@ -368,3 +406,102 @@ async def async_llm_wrapper():
 
         self._llm_engine = None
         logger.info("Shutdown complete")
+
+    async def _get_remote_prefill_response(self, request):
+        prefill_request = copy.deepcopy(request)
+        prefill_request.sampling_params["max_tokens"] = 1
+        prefill_request.disaggregated_params = DisaggregatedParams(
+            request_type="context_only"
+        )
+
+        if self._prefill_client is None:
+            raise ValueError("Prefill client not initialized")
+
+        # TODO: Use smart KV router to determine which prefill worker to use.
+        ctx_responses = [
+            ctx_response
+            async for ctx_response in await self._prefill_client.round_robin(
```

Comment:
question: does this use a prefill queue of workers?

Reply:
No. We directly send the requests to prefill workers in round-robin fashion currently.